crates/m3/tests/merkle_tree.rs

```diff
@@ -626,6 +642,22 @@
 					&trace.root.clone().into_iter().collect::<Vec<_>>(),
 				)
 				.expect("Failed to fill roots table");
+
+			let lookup_counts = tally(cs, witness, &[], self.lookup_channel, &IncrIndexedLookup)
+				.expect("Failed to tally lookup counts");
+
+			// Fill the lookup table with the sorted counts
+			let sorted_counts = lookup_counts
+				.into_iter()
+				.enumerate()
+				.sorted_by_key(|(_, count)| Reverse(*count))
+				.collect::<Vec<_>>();
+			witness
+				.fill_table_parallel(&self.incr_table, &sorted_counts)
+				.expect("Failed to fill incr lookup table");
```

Comment:
Raise error.

Expect should only be called with messages that explain why the called function _will not_ return an error. If it _may_ return an error, you have to propagate the error.